### PR TITLE
Display project overview 

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

**Comment by sven1103** [comment]
image

File: mariadb-connector/src/main/java/life/qbic/projectmanagement/persistence/ProjectJpaRepository.java [hunk 1]
```diff
@@ -40,6 +48,15 @@ public void add(Project project) {
     projectRepo.save(project);
   }
 
+  @Override
+  public List<ProjectPreview> getAllPreviews(int offset, int limit) {
+    List<ProjectPreview> projects = new ArrayList<>();
+    //todo how to handle projects with null values for e.g proejct objective?
```
Comment (wow-such-code) [inline mariadb-connector/src/main/java/life/qbic/projectmanagement/persistence/ProjectJpaRepository.java:54]: this is meant to stay in there? (just askin')

File: projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectPreview.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import java.util.Objects;
+
+/**
+ * A limited view of the more complex {@link Project}.
+ *
+ * @since 1.0.0
+ */
+@Entity
+@Table(name = "projects_datamanager")
+public class ProjectPreview {
```
Comment (wow-such-code) [inline projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectPreview.java:16]: Wasn't there another class doing the same?

Comment (sven1103) [reply projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectPreview.java:16]: please see the class' JavaDocs :)

File: mariadb-connector/src/main/java/life/qbic/projectmanagement/persistence/ProjectJpaRepository.java (hunk 1 above)
Comment (sven1103) [reply mariadb-connector/src/main/java/life/qbic/projectmanagement/persistence/ProjectJpaRepository.java:54]: nice catch
I will remove it

**Review by wow-such-code — APPROVED** [review]

**Pull request merged**